Trumpf.Coparoo.Web/Controls/Table/Body.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Trumpf.Coparoo.Web.Controls.Table
```

Comment:
Now if a tests writes using `Table` one also see `Body`, `Row`, `Segment`, `Cell` etc. which I somehow don't like because there is no obvious link to the table anymore. If you would move all these classes inside the `Table` class, it would be a bit like before (`Table.Cell` instead of `TableCell` etc.). In turn, the `Table` class could move into the `Controls` namespace, hence no need for the `Table` namepace anymore and hence more easily accessible to users. What do you think?

Reply:
Good suggestion. I renamed the folder to `HTMLTable` instead of `Table` so there are no conflicts
![image](https://user-images.githubusercontent.com/9483031/36948245-3bda81ea-1fd8-11e8-82c0-5f13951c3e3e.png)

Reply:
I would much prefer to have it directly in the `Table` class like this:
```
public class Table {
  public class Row {...}
  public class Cell {...}
  ...
  public class Segment {...}
  // the table properties etc.
```
If the class gets too large you can split it up using keyword `partial` ([details](https://docs.microsoft.com/de-de/dotnet/csharp/programming-guide/classes-and-structs/partial-classes-and-methods)). You can then even keep the current files, which would be nice. To that end, you could add `public partial class Table` before each of the other types.

Reply:
![image](https://user-images.githubusercontent.com/9483031/36951046-ffc3b056-1ffe-11e8-8c23-7d3cc17ce3c2.png)

Any suggestions for fixing?

Reply:
Maybe renaming the class files of the segments accordingly to their html counterpart? E.g. `Body.cs` becomes `TBody.cs`?

Reply:
What about naming the properties `Header`, `Content` and `Footer`? At least to former and latter keep the notation from [https://www.w3schools.com/html/html_tables.asp](). Or `Data` instead of `Content`.

Reply:
`Header`, `Content` and `Footer` sound fine to me